book/website/collaboration/ppie-events/ppie-events-location.md

```diff
@@ -0,0 +1,74 @@
+(cl-ppie-events-location)=
+# Location
+This section provides guidance that is most relevant for in-person or hybrid events. 
+
+The location of your event plays a pivotal role in supporting or hindering how accessible and inclusive your event will be for people with lived experience. As with all planning, reflect on how you will get input and feedback from your attendees with lived experience in order to identify the best venue for your event.
+
+
+(cl-ppie-events-choosing-venue)=
+## Choosing a Venue
+Before deciding on a venue it is recommended to have an organizer or member of the team visit potential venues to do a walk-through and check that all rooms in the venue will work for your event’s needs. 
+Include people with lived experience in this process as they can have expertise in identifying barriers such as inaccessibility for wheelchair users, poor lighting, or sensory overstimulation. 
+There is more guidance on how to choose an Accessible location in the Accessible Events Guide. 
+This can be helpful to both check the maintenance and layout of the space, and can also help you plan your event set-up such as where to put signs directing attendees to your event.
+
+Through a Universal Design lens, a venue that is able to support a diverse attendee list will be best for any event that involves both a research and general public audience. 
+
+When reviewing potential venues, consider the ventilation of airflow, lighting and acoustics, and ensure that they are appropriate for the number of attendees at your event. 
+A large hall with high ceilings may cause echoing and may not be suitable for discussion-based sessions. Similarly, a small room with limited ventilation may not be suited for long sessions with a large number of attendees. 
+These considerations will benefit all events, however may be more important to consider for events that are intended for people with lived experience. 
+It is important to ensure that a space can support the involvement of all its attendees. 
+This can be considering how to layout tables (circular is best) so that all those seated are able to listen and contribute equally to the discussion, and can also be considering how the space itself will be laid out. 
```

Comment:
```suggestion
This can be considering how to lay out tables (circular is best) so that all those seated are able to listen and contribute equally to the discussion, and can also be considering how the space itself will be laid out. 
```
Maybe also ensuring sufficient seating areas, as well as assigned quiet and/or sensory spaces?